build.js

```diff
@@ -44,12 +44,12 @@ const markedOptions = {
 // in DEFAULT_LANG as a fallback to prevent any strings that aren't filled out
 // from appearing as blank.
 function i18nJSON (lang) {
-  var defaultJSON = require(`./locale/${DEFAULT_LANG}/site.json`)
-  var templateJSON = require(`./locale/${lang}/site.json`)
-  var finalJSON = JSON.parse(JSON.stringify(defaultJSON))
-  var merge = function (targetJSON, customJSON) {
-    Object.keys(customJSON).forEach(function (key) {
-      let value = customJSON[key]
+  const defaultJSON = require(`./locale/${DEFAULT_LANG}/site.json`)
+  const templateJSON = require(`./locale/${lang}/site.json`)
+  const finalJSON = JSON.parse(JSON.stringify(defaultJSON))
+  const merge = (targetJSON, customJSON) => {
```

Comment:
I'd also like to see `Object.assign()` here. I'm okey with that change being a separate PR though.